tensorbay/dataset/dataset.py

```diff
@@ -51,7 +102,7 @@ def __init__(self, name: str, is_continuous: bool = False) -> None:
         super().__init__(name)
         self._segments: NameSortedList[_T] = NameSortedList()
         self._catalog: Catalog = Catalog()
-        self._is_continuous = is_continuous
+        self._notes = Notes(is_continuous)
```

Comment:
we should remove the `is_continuous` from the `__init__` of `Dataset`, in order make the creation of `Dataset` and `DatasetClient` consistent